src/components/RegionList/RegionListComponent.tsx

```diff
@@ -80,7 +80,8 @@ export class RegionListComponent extends React.Component<WidgetProps> {
 
         let nameWidth = RegionListComponent.NAME_COLUMN_DEFAULT_WIDTH;
         const availableWidth = this.width - 2 * padding;
-        let fixedWidth = RegionListComponent.TYPE_COLUMN_DEFAULT_WIDTH + RegionListComponent.CENTER_COLUMN_DEFAULT_WIDTH + RegionListComponent.SIZE_COLUMN_DEFAULT_WIDTH + RegionListComponent.ROTATION_COLUMN_DEFAULT_WIDTH;
+        // tslint:disable-next-line: max-line-length
```

Comment:
why not just put this on two lines, instead of disabling tslint for this line?

Reply:
Fixed